coffee_cmd/src/coffee_term/command_show.rs

```diff
@@ -97,7 +97,7 @@ pub fn show_nurse_verify(nurse_verify: &ChainOfResponsibilityStatus) -> Result<(
         table.push([
             term::format::dim(String::from("●")),
             term::format::bold(String::from("Defects")),
-            term::format::bold(String::from("Affected repositories")),
+            term::format::bold(String::from("Action")),
```

Comment:
```suggestion
            term::format::bold(String::from("Affected")),
```

thanks @tareknaser for the suggestion